config/config.go

```diff
@@ -114,10 +116,11 @@ func Parse(in *MountParams) (*MountConfig, error) {
 	}
 
 	out.PodInfo = &PodInfo{
-		Namespace:      attrib[attributePodNamespace],
-		Name:           attrib[attributePodName],
-		UID:            types.UID(attrib[attributePodUID]),
-		ServiceAccount: attrib[attributeServiceAccountName],
+		Namespace:            attrib[attributePodNamespace],
+		Name:                 attrib[attributePodName],
+		UID:                  types.UID(attrib[attributePodUID]),
+		ServiceAccount:       attrib[attributeServiceAccountName],
```

Comment:
nit: Given the mapping of `attributePodNamespace <-> Namespace` & `` can `ServiceAccount` be made `ServiceAccountName` for a more uniform naming convention (also can the order of declaring them be fixed, name them namespace then UID and so on)?

Reply:
Ordering resolved